README.md

```diff
@@ -73,39 +77,27 @@ pip install -r requirements/gui.txt
 After this, the command `python joinmarket-qt.py` from within the `scripts` subdirectory should work.
 There is a [walkthrough](docs/JOINMARKET-QT-GUIDE.md) for what to do next.
 
-### Notes on architectural changes (can be ignored)
+### Architecture notes
 
-Motivation: By separating the code which manages conversation with other
-Joinmarket participants from the code which manages this participant's Bitcoin
-wallet actions, we get a considerable gain at a minor cost of an additional layer:
-code dependencies for each part are much reduced, security requirements of the 
-server/daemon layer are massively reduced (which can have several advantages such as
-it being more acceptable to distribute this layer as a binary), and client code
-can be written, implementing application-level logic (do join with coins X under condition X)
-using other Bitcoin libraries, or wallets, without knowing anything about
-Joinmarket's inter-participant protocol. An example is my work on the Joinmarket
-electrum [plugin](https://github.com/AdamISZ/electrum-joinmarket-plugin).
+See [architecture-notes.md](docs/architecture-notes.md).
```

Comment:
Seems reasonable, since most readers don't need this. It does remind me that we could add a note to the wallet features list that explains that the message daemon can run on a separate machine, but to add that i think we'd also need a short guide on the right way to do it, so, definitely not something to address here.